packages/react-components/react-portal/library/src/components/Portal/usePortalMountNode.ts

```diff
@@ -134,7 +134,8 @@ const useModernElementFactory: UseElementFactory = options => {
         // before mounting the portal. We hardcode the value to `Node.ELEMENT_NODE` to pass this check and avoid
         // premature node creation
         if (property === 'nodeType') {
-          return Node.ELEMENT_NODE;
+          // Can't use the `Node.ELEMENT_NODE` as it's a browser API and  not available in all environments, e.g SSR
```

Comment:
we need to automate this -> lets add `NODE` to restrictedGlobals `packages/eslint-plugin/src/shared/restricted-globals.js`

Reply:
https://github.com/microsoft/fluentui/issues/35356